rllib/examples/_old_api_stack/connectors/run_connector_policy.py

```diff
@@ -24,34 +28,45 @@ def run(checkpoint_path, policy_id):
 
     # Run CartPole.
     env = gym.make("CartPole-v1")
+    env_id = "env_1"
     obs, info = env.reset()
-    terminated = truncated = False
-    step = 0
-    while not terminated and not truncated:
-        step += 1
-
+    # Run for 2 episodes.
+    episodes = step = 0
+    while episodes < 2:
         # Use local_policy_inference() to run inference, so we do not have to
         # provide policy states or extra fetch dictionaries.
         # "env_1" and "agent_1" are dummy env and agent IDs to run connectors with.
         policy_outputs = local_policy_inference(
```

Comment:
local_policy_inference is a monster from the ancient times :) But it shows us what a single action needed back then. The single action inference in the wild is sth we have to think about, too.